src/middlewared/middlewared/etc_files/loader.py

```diff
@@ -28,7 +28,11 @@ def generate_loader_config(middleware):
 
     config = []
     for generator in generators:
-        config.extend(generator(middleware) or [])
+        try:
+            config.extend(generator(middleware) or [])
+        except Exception as e:
+            middleware.logger.error("Failed to load generator with error: %s", e)
```

Comment:
We need to know which `generator`

`middleware.logger.error("Failed to load generator %r with error: %s", generator, e)`